tipsNtricks.md

```diff
@@ -182,14 +192,17 @@
 27. **LIfe360** - Locate your family and friends using GPS
 28. **AdAway** - Block most of Ads on your phone
 29. **SplitWise** - Split bills and expenses with friends
+30. **ES File Explorer** - Share file b/w android and laptop ,arrange file, compress files and lot more.
+31. **WiFi Mouse | keyboard** - Transform phone into a wireless mouse, keyboard and trackpad using WiFi Mouse.
+32. **Super Backup** - Get your installed apps apk. backup apps & contacts & SMS & Call logs & Bookmarks & Calendars to the SD card/Google Drive/Gmail.
```

Comment:
Doesn't ES File Explorer do this. I am sure about the apps not about contacts, SMS and calls.